### ISSUE TITLE
Is longformer pre trained to work for text summarizarion?

### ISSUE THREAD

**Issue by gandharvsuri** [allenai/longformer#25]

**Comment by ibeltagy** [comment]
It is pretrained on masked language modeling, and not for any specific downstream task, for that you still need to finetune it on the task-specific data. Think of it as BERT but works for long docs.

**Comment by ibeltagy** [comment]
Closing this issue in favor of https://github.com/allenai/longformer/issues/28
